scripts/irods/database_connect.py

```diff
@@ -505,11 +505,12 @@ def get_next_object_id():
             timestamp)
 
     if default_resource_directory:
-        default_resc_id = get_next_object_id()
-        execute_sql_statement(cursor,
+        if default_resource_name:
```

Comment:
Perhaps a comment to explain what it means when `default_resource_name` is missing (no resource attached to this server)?

Reply:
yeah, this logic isn't quite baked... i think there's a missing box in the matrix... but not sure yet.